protocols/kad/src/behaviour.rs

```diff
@@ -884,6 +925,13 @@ where
     ///
     /// > **Note**: Bootstrapping requires at least one node of the DHT to be known.
     /// > See [`Behaviour::add_address`].
+    ///
+    /// > **Note**: Bootstrap does not require to be called manually. It is periodically
+    /// invoked at regular intervals based on the configured `periodic_bootstrap_interval` (see
+    /// [`Config::set_periodic_bootstrap_interval`] for details) and it is also automatically invoked
+    /// when a new peer is inserted in the routing table.
+    /// These two config parameters are used to call [`Behaviour::bootstrap`] periodically and automatically
```

Comment:
there seems to only be referenced one config parameter on this note, `Config::set_periodic_bootstrap_interval` right?

Reply:
Fixed! I added a reference for `automatic_bootstrap_throttle` as well!

Reply:
Indeed @jxs, I forgot to change the documentation after Thomas suggestion : https://github.com/libp2p/rust-libp2p/pull/4838#discussion_r1489026442

> I think your point is valid and I am happy to add such a throttling but I'd like to do so without a configuration value so we can more easily change this later.

Thomas preferred that we did not make throttling configurable at first in order to easily remove it or modify it in the future without releasing a breaking change if it were needed.

So if this still stands, I don't know if it is OK to reference a private method in a public documentation ?

Reply:
yeah I'd just omit that part and mention the only configurable parameter exposed to the end user